docs/source/installation.mdx

```diff
@@ -51,10 +51,9 @@ You'll need to build bitsandbytes from source. To compile from source, you need
 
 ```bash
 git clone https://github.com/TimDettmers/bitsandbytes.git && cd bitsandbytes/
-pip install -r requirements-dev.txt
 cmake -DCOMPUTE_BACKEND=cuda -S .
 cmake --build . --config Release
-python -m build --wheel
+pip install .
```

Comment:
This is a different thing. `python -m build --wheel` will build a wheel and not install it. `pip install .` builds a throwaway wheel and installs it.

Reply:
> In general though I think the docs on that page could be expanded a little bit and offer more information.

Yes, I think that would be good. Do you have anything in specific in mind that you feel must be included?

Reply:
A few things that come to mind that I may be able to contribute to:
* Explain there's separate builds for NO_CUBLASLT, CUDA, and CPU-only
** _Aside_ I'm wanting to get rid of the need for the NO_CUBLASLT build, future PR may happen for that.
* Specify compilers we know to work
** E.g. for MSVC, we've been able to build with VS2022 (MSVC 193X), but as far as I am aware have not validated older versions.
** To that point, we haven't built against a lot of CUDA toolkit versions yet. CUDA 11.6 appears to be the minimum for VS2022
** For gcc on x86-64, the bare minimum is probably 6.x?
** I haven't verified but I think MSVC 1934 will be required for 12.0+.
* Should maybe explain how to set up build environment on Windows, with the [Developer Command Prompt](https://learn.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022) probably being easiest.
* Specify some hardware constraints on CPU: e.g. we expect AVX2 support on x86-64 (at least for Windows build, but would expect the same on Linux in the future, i.e. we expect minimum `x86-64-v2`)
* Minimum OS requirements
** For Windows, this would be Windows Server 2019+, Windows 10+, x86-64 only.
** Not clear to me for macOS, but still WIP

Reply:
Reference for VS support

![Reference](https://upload.wikimedia.org/wikipedia/commons/thumb/3/37/CUDA_Visual_Studio_matrix.jpg/1024px-CUDA_Visual_Studio_matrix.jpg)